JSONModel/JSONModel/JSONModelCustomSetter.h

```diff
@@ -0,0 +1,18 @@
+//
+//  JSONModelCustomSetter.h
+//  JSONModel
+//
+//  Created by Dmytro Povolotskyi on 09/02/2017.
+//  Copyright © 2017 com.jsonmodel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface JSONModelCustomSetter : NSObject
+
+@property ( nonatomic, readonly, strong) NSValue* value;
+@property ( nonatomic, readonly, assign) BOOL withErrorOutParam;
+
+- (instancetype)initWithValue:(NSValue*)value withErrorOutParam:(BOOL) withErrorOutParam;
```

Comment:
Change to:

```objc
- (instancetype)initWithSelector:(SEL)selector;
```

Determine the presence of the error param based on the selector - don't pass this in